scalajs/src/main/scala/clue/js/WebSocketJSBackend.scala

```diff
@@ -34,61 +35,60 @@ final class WebSocketJSBackend[F[_]: Async: Logger] extends WebSocketBackend[F]
       isOpen     <- Ref[F].of(false)
       isErrored  <- Ref[F].of(false)
       connection <-
-        Dispatcher[F].use { dispatcher =>
```

Comment:
Ok, I think we had some discussion whether to pass the dispatcher or create it in place but it was never clear the reasons for one or the other